Lib/LibStatic/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ IF(WIN32 AND NOT UNIX)
 
 	IF(NOT ${CMAKE_GENERATOR} STREQUAL "MSYS Makefiles")
 
-		IF( MSVC10 OR MSVC11 OR MSVC12 )
+		IF( MSVC10 OR MSVC11 OR MSVC12 OR MSVC14 )

```

Comment:
Seems like a duplicate of this PR: https://github.com/OculusVR/RakNet/pull/64
which I commented. The proper way to do it here, would probably be:
```
if (MSVC_VERSION GREATER_EQUAL 1600)
```